client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -149,6 +149,7 @@ const PatternAssembler = ( {
 				font_variation_title: getVariationTitle( fontVariation ),
 				font_variation_type: getVariationType( fontVariation ),
 				assembler_source: getAssemblerSource( selectedDesign ),
+				has_premium_styles_selected: numOfSelectedGlobalStyles > 0,
```

Comment:
[nit] Keep the term "global styles" for consistency:
```suggestion
				has_global_styles_selected: numOfSelectedGlobalStyles > 0,
```